docs/source/redhat-opens-install.rst

```diff
@@ -131,17 +123,126 @@ On a control node, perform the following steps:
    -  Find the lines for ``api_workers`` and ``rpc_workers``, uncomment
       them and set them both to 0.
 
-6. Restart the neutron server process:
-   ``service neutron-server restart``.
+5. Install and configure etcd. For these instructions we assume you're
+   configuring a single-node cluster on only one machine. If you plan to
+   deploy a multi-node cluster, please consult the `etcd clustering docs`_, and
+   skip to step 7.
+
+   - Download, unpack, and install the binary::
+
+        curl -L  https://github.com/coreos/etcd/releases/download/v2.0.9/etcd-v2.0.9-linux-amd64.tar.gz -o etcd-v2.0.9-linux-amd64.tar.gz
+        tar xvf etcd-v2.0.9-linux-amd64.tar.gz
+        cd etcd-v2.0.9-linux-amd64
+        mv etcd* /usr/local/bin/etcd
+
+   - Create an etcd user::
+
+        adduser -s /sbin/nologin -d /var/lib/etcd/ etcd
+        chmod 700 /var/lib/etcd/
+
+   - Mount a ramdisk at /var/lib/etcd::
+
+        sudo mount -t tmpfs -o size=512m tmpfs /var/lib/etcd
+
+   - Add the following to the bottom of ``/etc/fstab`` so that the ramdisk gets
+     reinstated at boot time::
+
+        tmpfs /var/lib/etcd-rd tmpfs nodev,nosuid,noexec,nodiratime,size=512M 0 0
+
+   - Get etcd running by providing an init file.
+
+     For RHEL 6.5, place the following in ``/etc/init/etcd.conf``, replacing
+     ``<hostname>`` and ``<public_ip>`` with their appropriate values for the
+     machine. For ``<cluster_id>`` it can be any textual string, but make sure
+     to use a new one each time.
```

Comment:
Wording is a bit clumsy ('Use any string for <cluster_id>' would be clearer), and 'each time' is a bit ambiguous - it should be unique to the cluster but then fixed I think?